src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -3631,12 +3631,6 @@ int getOrCreateSlot(int containingSlot, Symbol symbol)
             }
         }
 
-        [Obsolete("Use VisitCollectionElementInitializer(BoundCollectionElementInitializer node, TypeSymbol containingType, bool delayCompletionForType) instead.", true)]
-        private new void VisitCollectionElementInitializer(BoundCollectionElementInitializer node)
```

Comment:
>VisitCollectionElementInitializer [](http://example.com/codeflow?start=25&length=33)

It looks like this method was added intentionally, to catch cases where a specific method should be used instead.

Reply:
@cston I added it back, but I don't get what's the problem removing it. The `Obsolete` attribute will catch cases where this method shouldn't be used. But deleting it will also ensure that it's not used (because it doesn't exist in the first place)

I'd like to understand more what kind of issues can happen if this is removed. Thanks!

Reply:
This method hides the base class method with the same signature. I believe this method definition is intended to prevent calling the base method.